src/components/ControlledTextInputField/ControlledTextInputField.js

```diff
@@ -26,12 +26,22 @@ const ControlledTextInputField = ({
   name,
   label,
   placeholder,
-  rules,
+  customValidationFunction,
```

Comment:
`customValidationFunctions`, again with the naming. This is an object that can contain multiple functions